### PR TITLE
v5.5 Desktop app changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Review by devinbinnie — APPROVED** [review]

**Review by marianunez — APPROVED** [review]

**Review by cpoile — APPROVED** [review]

File: source/install/desktop-app-changelog.rst [hunk 1]
```diff
@@ -6,19 +6,70 @@ Desktop application changelog
...
+**Release Day: September 15, 2023**
+
+**Download Binaries:** `Mattermost Desktop on GitHub <https://github.com/mattermost/desktop/releases/latest>`_
+
+**Note:** Mattermost v5.5.0 contains a medium severity level security fix. Upgrading is highly recommended. Details will be posted on our `security updates page <https://mattermost.com/security-updates/>`__ 30 days after release as per the `Mattermost Responsible Disclosure Policy <https://mattermost.com/security-vulnerability-report//>`__.
+
+Compatibility
+~~~~~~~~~~~~~~~
+
+- Desktop App is supported on any supported Extended Support Release or a newer Mattermost server version.
```
Comment (cwarnermm) [inline source/install/desktop-app-changelog.rst:28]: Curious: 
- Is the Desktop App really supported on _any_ ESR release? Or just the most recent ones? 
- Do we have a clear sense of what a "newer MM server version" indicates/suggests?

File: source/install/desktop-app-changelog.rst [hunk 2]
```diff
@@ -6,19 +6,70 @@ Desktop application changelog
...
+^^^^^^^^^^^^^
+
+- Fixed a crash in diagnostics when the server was unreachable.
+- Fixed bad user feedback on the server URL validation when plugins were disabled.
+- Fixed an issue where auto-updating the app wouldn't be properly disabled.
+
+Known Issues
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Users seeing an endless "Loading..." screen when attempting to log in to the app may need to manually remove their cache directory. For MacOS it is located in ``/Users/<username>/Library/Containers/Mattermost/Data/Library/Application Support/Mattermost`` and for Windows it is located in ``Users/<username>/AppData/Roaming/Mattermost``.
```
Comment (cwarnermm) [inline source/install/desktop-app-changelog.rst:57]:
```suggestion
- Users seeing an endless "Loading..." screen when attempting to log in to the app may need to manually delete their cache directory. For MacOS it is located in ``/Users/<username>/Library/Containers/Mattermost/Data/Library/Application Support/Mattermost`` and for Windows it is located in ``Users/<username>/AppData/Roaming/Mattermost``.
```

Comment (cwarnermm) [reply source/install/desktop-app-changelog.rst:57]: Can we also include the path for Linux OS users?

File: source/install/desktop-app-changelog.rst (hunk 1 above)
Comment (amyblais) [reply source/install/desktop-app-changelog.rst]: I edited it to "... any currently supported Mattermost server version".

File: source/install/desktop-app-changelog.rst (hunk 2 above)
Comment (amyblais) [reply source/install/desktop-app-changelog.rst:57]: Maybe @devinbinnie can help with that.

**Review by johndavidlugtu — APPROVED** [review]

File: source/install/desktop-app-changelog.rst (hunk 2 above)
Comment (devinbinnie) [reply source/install/desktop-app-changelog.rst:58]: For Linux it should be `~/config/Mattermost`, where ~ is the home directory.

**Review by devinbinnie — APPROVED** [review]

**Pull request merged**